<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLogin</class>
 <widget class="QWidget" name="CLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhLatinOnly|Qt::ImhUppercaseOnly</set>
  </property>
  <widget class="QPushButton" name="ButtonLogIn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditUsername">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditPassword">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labeLEmail">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLink">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Registration here</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
